tests/behaviour/malicious-calldata_test.py

```diff
@@ -1,109 +0,0 @@
-import tempfile
```

Comment:
Why is this file deleted?..

Reply:
Because it was written in a way that was completely incongruent with the style of all the other tests.

Reply:
But it needs to be there, it tests important things?.. The whole Uint256 library is based on the premise that it's given valid Uint256's, so we need to check for that. It's written that way, because unlike other tests it tests failure.

Reply:
It also uses fixtures, in order to avoid computing the same things again and again

Reply:
Fair enough, I'll add it back :).